pages/api/sitemap.js

```diff
@@ -0,0 +1,62 @@
+import { keyFigures } from "../../constants/key-figures";
+import {timelineOptions} from "../../constants/timeline-options";
+import * as fs from "fs";
+
+const absoluteLink = (path) => { return `https://blackwomenssuffrage.dp.la${path}` }
+
+const buildEntry = (link, date) => {
+    return `<url>
+    <loc>${link}</loc>
+    <lastmod>${date}</lastmod>
+</url>
+`;
+}
+const sitemap = (req, res) => {
+
+    const date = new Date().toISOString();
+
+    const staticLinks = [
+        '/',
+        '/about',
+        '/timeline',
+        '/collections',
+        '/collections/ida-b-wells',
+        '/partners'
+    ].map(absoluteLink);
```

Comment:
Do you want to add the `harmful-language-statement` path as well?

Reply:
crap, yes!